src/order/closure.lean

```diff
@@ -112,12 +127,45 @@ set.ext $ λ x, ⟨λ h, ⟨x, h⟩, by { rintro ⟨y, rfl⟩, apply c.idempoten
 /-- Send an `x` to an element of the set of closed elements (by taking the closure). -/
 def to_closed (x : α) : c.closed := ⟨c x, c.closure_is_closed x⟩
 
+lemma mk₃.to_closed {f : α → α} {p : α → Prop} {hf : ∀ x, x ≤ f x} {hfp : ∀ x, p (f x)}
+  {hmin : ∀ ⦃x y⦄, x ≤ y → p y → f x ≤ y} {x : α} {hx : p x} :
+  (mk₃ f p hf hfp hmin).closed x :=
+le_antisymm (hmin (le_refl _) hx) (hf _)
+
```

Comment:
I don't understand the purpose of this lemma, isn't it just a special case of the `to_closed` lemma on the line above?

Reply:
Oh, I see why I'm confused:
```suggestion
lemma mem_mk₃_closed {f : α → α} {p : α → Prop} {hf : ∀ x, x ≤ f x} {hfp : ∀ x, p (f x)}
  {hmin : ∀ ⦃x y⦄, x ≤ y → p y → f x ≤ y} {x : α} (hx : p x) :
  x ∈ (mk₃ f p hf hfp hmin).closed :=
le_antisymm (hmin (le_refl _) hx) (hf _)

```

I missed the x because `closed` is a set so should be using `∈`